CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Implemented Brave News to display inline ads. ([#3872](https://github.com/brave/brave-ios/issues/3872))
```

Comment:
nit: we can probably just use the same wording desktop did so it's the same across the board:

```
- Added display ads to the Brave News feed.
```

The one on desktop is a bit easier to understand re: not mentioning "inline".